src/components/molecules/Chatbox/components/ChatboxOptionsControls/ChatboxOptionsControls.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo } from "react";
+import { Dropdown, DropdownButton } from "react-bootstrap";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { ChatMessageOptions, ChatOption } from "types/chat";
+
+import { TextButton } from "components/atoms/TextButton";
+
+import "./ChatboxOptionsControls.scss";
+
+export interface ChatboxOptionsControlsProps {
+  activeOption?: ChatOption;
+  setActiveOption: React.Dispatch<React.SetStateAction<ChatOption | undefined>>;
+}
+
+export const ChatboxOptionsControls: React.FC<ChatboxOptionsControlsProps> = ({
+  activeOption,
+  setActiveOption,
+}) => {
+  const shouldShowPoll = activeOption === ChatMessageOptions.poll;
+
+  const dropdownOptions = useMemo(
+    () =>
+      Object.values(ChatMessageOptions).map((option) => (
+        <Dropdown.Item
+          key={option.name}
+          onClick={() => setActiveOption(option)}
+        >
+          {option.name}
+          <FontAwesomeIcon icon={option.icon} />
+        </Dropdown.Item>
+      )),
+    [setActiveOption]
+  );
+
+  return (
+    <div className="ChatboxOptionsControls">
+      {shouldShowPoll ? (
+        <TextButton
+          label="Cancel Poll"
+          onClick={() => setActiveOption(undefined)}
```

Comment:
useCallback

Reply:
fixed in 8d4c8419ab1cc372e7ac9490a1c01f585179c77c